test/op_shape_test.cpp

```diff
@@ -1483,14 +1483,14 @@ TEST_CASE(test_squeeze_wrong_axis)
 TEST_CASE(test_unsqueeze)
 {
     migraphx::shape s1{migraphx::shape::float_type, {4, 3, 3}};
-    migraphx::shape s2{migraphx::shape::float_type, {4, 3, 1, 3}};
+    migraphx::shape s2{migraphx::shape::float_type, {4, 3, 1, 3}, {9, 3, 3, 1}};
```

Comment:
Why do we need this change here? If you only put the lens in the shape, it will be a standard shape.

Reply:
Yes, not needed. Made the change.